torch/csrc/jit/passes/quantization.cpp

```diff
@@ -207,10 +209,15 @@ bool alwaysRaisesException(Block* block) {
   return false;
 }
 
-bool isAddScalar(Node* n) {
-  return (n->kind() == Symbol::aten("add") ||
-          n->kind() == Symbol::aten("add_")) &&
-      n->input(0)->type()->isSubtypeOf(TensorType::get()) &&
+bool hasScalarInput(Node* n) {
+  bool result = false;
+  std::unordered_set<std::string> scalar_op{"add", "add_", "mul", "mul_"};
+  for (const auto& op : scalar_op) {
+    if (n->kind() == Symbol::aten(op)) {
+      result |= true;
+    }
```

Comment:
this can be `result |= n->kind() == Symbol::aten(op)`

Reply:
actually, I think it's better to do find: https://github.com/pytorch/pytorch/blob/master/torch/csrc/jit/passes/quantization.cpp#L179

Reply:
you can refactor this part to a separate function and call it here: https://github.com/pytorch/pytorch/blob/master/torch/csrc/jit/passes/quantization.cpp#L171-L181